pkg/process/procutil/process_linux.go

```diff
@@ -611,10 +611,13 @@ func (p *probe) parseStatContent(statContent []byte, sInfo *statInfo, pid int32,
 		sInfo.ppid = int32(ppid)
 	}
 
-	flags, err := strconv.ParseInt(flagStr, 10, 32)
+	flags, err := strconv.ParseUint(flagStr, 10, 32)
```

Comment:
`flags` is uint32 and `math.MaxInt32` is smaller than `math.MaxUint32`. Ref: https://pkg.go.dev/math#pkg-constants

So, this change shouldn't be a breaking change.

https://github.com/DataDog/datadog-agent/blob/73867119465d5e492d7674f222172bd800108842/pkg/process/procutil/process_linux.go#L56-L62

Reply:
Ref: https://man7.org/linux/man-pages/man5/proc.5.html#:~:text=the%20process.%0A%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20(9)-,flags,-%25u%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20The%20kernel

```
              (9) flags  %u
                     The kernel flags word of the process.  For bit
                     meanings, see the PF_* defines in the Linux kernel
                     source file include/linux/sched.h.  Details depend
                     on the kernel version.

                     The format for this field was %lu before Linux 2.6.
```